core/modules/macros/contrastcolour.js

```diff
@@ -29,6 +29,22 @@ exports.params = [
 Run the macro
 */
 exports.run = function(target,fallbackTarget,colourA,colourB) {
```

Comment:
I guess it's really not necessary to have all these parameters to the macro any longer, target would be entirely sufficient... or rather "color".